src/interpreter.cpp

```diff
@@ -586,6 +582,7 @@ struct command_info cmd_info[] =
     { "close"      , POS_SITTING , do_gen_door , 0, SCMD_CLOSE, BLOCKS_IDLE_REWARD },
     { "closecombat", POS_LYING   , do_closecombat, 0, 0, BLOCKS_IDLE_REWARD },
     { "cls"        , POS_DEAD    , do_gen_ps   , 0, SCMD_CLEAR, ALLOWS_IDLE_REWARD },
+    { "cook"       , POS_LYING   , do_cook     , 0, 0, BLOCKS_IDLE_REWARD },
```

Comment:
Why de-alphabetize this from after `complete`? Also, you've removed quite a few other normal commands (`copy`, `delete`, `download`, `software`, `upload`) and matrix commands (`decrypt`, `disarm`, `download`, `load`, `unload`, `software`, `memory`), where did their functionality go?

Reply:
I forgot how the alphabet works. Their functionality will be implemented later.